view.rb

```diff
@@ -116,4 +170,16 @@ def self.account_info(acct_list)
     end
     account_id_array
   end
+
+  def self.account_info2(acct_list)
```

Comment:
Could use a more effective display for displaying the total list of current users.